accounts-db/src/accounts.rs

```diff
@@ -61,7 +86,18 @@ impl AccountLocks {
     }
 
     fn is_locked_write(&self, key: &Pubkey) -> bool {
-        self.write_locks.contains(key)
+        self.write_locks.get(key).map_or(false, |count| *count > 0)
```

Comment:
it should not be possible for count to be 0, right?

Reply:
I see code is same for readonly, so it's fine. Just a bit odd since this will never be different than simple `contains`